specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/networkWatcher.json

```diff
@@ -3192,6 +3192,10 @@
           "description": "ID of the storage account which is used to store the flow log.",
           "type": "string"
         },
+        "enabledFilteringCriteria": {
```

Comment:
I still have some doubts on the clarity of calling this 'enabledFilteringCriteria', but the description is certainly much improved now, thanks.

Reply:
I think it would still be good if the documentation of this field either linked to the docs of the minilanguage, or included enough examples so that a supplementary document is mostly redundant.

Reply:
Thanks for the feedback. We will add the link to the docs once the documentation is made public. Will do this in upcoming versions.